src/components/ui/TripCard/AccordianDetails/DetailsGrid/DetailItem/DetailItem.tsx

```diff
@@ -0,0 +1,33 @@
+import { Stack, Typography } from '@mui/material';
+import { DetailsItemWrapper } from './DetailItem.styled';
+
+interface IDetailsItemProps {
+    title: string;
+    value: string;
+    imageSrc: string;
+    imageAlt: string;
+}
+export const DetailsItem = ({
+    title,
+    value,
+    imageSrc,
+    imageAlt,
+}: IDetailsItemProps) => {
+    return (
+        <DetailsItemWrapper direction={'row'} justifyContent={'space-between'}>
```

Comment:
Suggesting that the content be left aligned for readability.

Reply:
Resolved in 209caa9248812fc845de2ab97206e85fa7cbece5